barretenberg/cpp/src/barretenberg/client_ivc/client_ivc.test.cpp

```diff
@@ -54,12 +54,6 @@ class ClientIVCTests : public ::testing::Test {
         Builder circuit{ ivc.goblin.op_queue };
         MockCircuits::construct_arithmetic_circuit(circuit, log2_num_gates);
 
-        // TODO(https://github.com/AztecProtocol/barretenberg/issues/911): We require goblin ops to be added to the
```

Comment:
moved this inside ClientIvc to avoid needing to handle it in multiple locations. Made new more descriptive issue.